src/pybind/mgr/dashboard/plugins/ses7p_motd.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import hashlib
+import json
+
+from mgr_module import Option
+
+from . import PLUGIN_MANAGER as PM
+from . import interfaces as I  # noqa: E741,N812
+
+
+@PM.add_plugin  # pylint: disable=too-many-ancestors
+class SES7PMotd(I.CanMgr, I.Setupable, I.HasOptions):
+    @PM.add_hook
+    def get_options(self):
+        return [Option(
+            name='ses7p_motd_configured',
+            default=False,
+            type='bool',
+            desc='Is the MOTD for SES7P configured')]
+
+    @PM.add_hook
+    def setup(self):
+        # Check whether the MOTD has already been configured. This is
+        # done to ensure the SES7P announcement is not configured on
+        # every start of the Ceph Dashboard module and to allow the
+        # administrator to disable it if necessary.
+        configured = self.mgr.get_module_option('ses7p_motd_configured')
+        if configured:
+            return
+        # Note, we can not call the 'ceph dashboard motd set ...' command
+        # here because it is not available at this time. To workaround
+        # this issue we build and write the MOTD configuration ourself.
+        message = 'There is an upgrade to SES7P available, please check ' \
+                  'https://documentation.suse.com/ses/7p/html/ses-all/ses-upgrade.html ' \
+                  'for more information.'
+        value: str = json.dumps({
+            'message': message,
+            'md5': hashlib.md5(message.encode()).hexdigest(),
+            'severity': 'warning',
```

Comment:
Do we want to allow the user to hide the MOTD in the UI or do we want to show it permanently. In the latter case the MOTD can only be disabled via CLI by the administrator.

Reply:
Without being able to give a definitive answer, it's the administrator we want to be informed about that upgrade, right?

Reply:
> Without being able to give a definitive answer, it's the administrator we want to be informed about that upgrade, right?

Yes, but all users will get the MOTD because this feature does not provide the feature to selective display it to a configured user or role.

Reply:
I'm afraid it's not possible, but I'll add my initial thought nevertheless. I think it would be great if we can at least show the message once per session. If the user hides the message it won't be gone forever. The message should be shown again as soon as the user does a re-login.

If the idea above it not possible, I'm not in favor of giving the option to hide the message.